server/src/test/java/org/opensearch/action/main/MainResponseTests.java

```diff
@@ -98,7 +100,8 @@ public void testToXContent() throws IOException {
                     + "\"build_snapshot\":" + current.isSnapshot() + ","
                     + "\"lucene_version\":\"" + version.luceneVersion.toString() + "\","
                     + "\"minimum_wire_compatibility_version\":\"" + version.minimumCompatibilityVersion().toString() + "\","
-                    + "\"minimum_index_compatibility_version\":\"" + version.minimumIndexCompatibilityVersion().toString() + "\"}"
+                    + "\"minimum_index_compatibility_version\":\"" + version.minimumIndexCompatibilityVersion().toString() + "\"},"
+                + "\"tagline\":\"" + TAGLINE + "\""
```

Comment:
nit: indentation

Reply:
Thanks a lot for your review! 😄 👍
About the indentation, it was the same when the "tagline" was not removed.  (https://github.com/tlfeng/OpenSearch/blob/83e87f7e541bf1153645762261f54c73bb766ee6/server/src/test/java/org/elasticsearch/action/main/MainResponseTests.java#L87)